docs/tutorials/cups/01_bootstrap_machines.md

```diff
@@ -0,0 +1,94 @@
+# 1. Bootstrap all Machines
+
+
+This section covers how to install Juju and MicroK8s to act as the infrastructure for SD-Core, and may be skipped as needed if already done.
+
+## SD-Core Control Plane
+
+All commands in this section are run on the SD-Core Control Plane Kubernetes cluster host.
+
+Install MicroK8s and configure MetalLB to expose 1 IP address for the AMF (`10.201.0.201`) and one for the Network Management Service (`10.201.0.202`):
+
+```console
+sudo snap install microk8s --channel=1.27-strict/stable
+sudo microk8s enable hostpath-storage
+sudo microk8s enable metallb:10.201.0.201-10.201.0.202
+```
+
+Export the Kubernetes configuration and copy that to the controller:
+
+```console
+sudo microk8s.config > control-plane-cluster.yaml
+scp control-plane-cluster.yaml juju-controller.mgmt:
+```
+
+## SD-Core User Plane
+
+All commands in this section are run on the SD-Core User Plane Kubernetes cluster host.
+
+Install MicroK8s, configure MetalLB to expose 1 IP address for the UPF, and add the Multus plugin:
+
+```console
+sudo snap install microk8s --channel=1.27-strict/stable
+sudo microk8s enable hostpath-storage
+sudo microk8s enable metallb:10.201.0.151-10.201.0.151
+sudo microk8s addons repo add community \
+    https://github.com/canonical/microk8s-community-addons \
+    --reference feat/strict-fix-multus
+sudo microk8s enable multus
+```
+
+Export the Kubernetes configuration and copy that to the controller:
+
+```console
+sudo microk8s.config > user-plane-cluster.yaml
+scp user-plane-cluster.yaml juju-controller.mgmt:
+```
+
+In this guide, the following network interfaces are available on the SD-Core User Plane machine:
+
+| Interface Name | Purpose |
+|----------------|---------|
+| ens3           | internal Kubernetes management interface.  This maps to the `management` subnet and must have an IP address |
+| ens4           | access interface.  This maps to the `access` subnet, and does not have an IP on the host |
+| ens5           | core interface.  This maps to the `core` subnet, and does not have an IP on the host.  Note that internet egress is required here and routing tables must be set to route gNB generated traffic |
+
+Now we create the MACVLAN bridges for ens4 and ens5, and label them accordingly:
+
+```console
+sudo ip link add access link ens4 type macvlan mode bridge
+sudo ip link set dev access up
+sudo ip link add core link ens5 type macvlan mode bridge
+sudo ip link set dev core up
+```
+
+## Juju Controller
+
+Begin by installing MicroK8s to hold the Juju controller, and configure MetalLB to expose one IP address for the controller (10.201.0.101), and one for the Canonical Observability Stack (10.201.0.102).
```

Comment:
Do we want COS integration to be part of this tutorial? Since it's optional, to me it seems like a good topic for a separate document.

Reply:
The idea is that we can see the full end to end, but I don't know.  I was basing it off the first tutorial, so not sure.

What do others think?

Reply:
As agreed in the meeting today, we will keep it in as part of the mastering concept.